src/layouts/dashboardLayout/dashboardLayout.module.css

```diff
@@ -24,5 +33,45 @@
   border-left-style: none;
   border-right-style: none;
   border-bottom: 1px solid #ccc;
-  margin-bottom: 32px;
+  margin-bottom: 16px;
+}
+
+.hiddenDivider {
```

Comment:
It was easier to leave the divider there, hide it, and slightly change the styling than it was to change the styling of other elements if it was set to `display: none`.